frontend/apps/marketing/src/components/iframe/iframeContentfulDefinition.ts

```diff
@@ -0,0 +1,48 @@
+// Creates a definition for the Iframe component to be used in Contentful Studio
+import {ComponentDefinition} from '@contentful/experiences-sdk-react';
+
+export const IframeContentfulComponentDefinition: ComponentDefinition = {
+  id: 'iframe',
+  name: 'iFrame Block',
+  category: 'Advanced',
+  thumbnailUrl:
+    'https://images.ctfassets.net/90t6bu6vlf76/1qy9FC9Bqb4ADrpyszIa5M/eb4c9dde9c1c90ab40036e8fa4412697/component_iframe_thumbnail.png',
+  tooltip: {
+    description:
+      'Embed external content using an iframe. Ideal for embedding videos, interactive forms, or third-party widgets within a page.',
```

Comment:
Still seeing videos here, but [this commit](https://github.com/code-dot-org/code-dot-org/pull/64496/commits/df3de5b1ddb30a647f20dc5d2eee2545ae52ccfe) is updated 🤔